### PR TITLE
Bugfix/fix downsampling oversampling secondary

### PR DESCRIPTION
**Pull request by jsnedecor**
The current script doesn't handle bams with a lot of secondary alignments correctly. It oversamples from chromosomes that have a high number of secondary alignments.

### CODE REVIEW

File: zippy/downsampling_bloom.py [hunk 1]
```diff
@@ -144,19 +144,16 @@ def run(self):
...
     perChromCount = {}
-    stats = pysam.idxstats(bam)
-    for line in stats.split('\n'):
-        tokenized = line.split()
-        if len(tokenized) == 0 : continue
-        if not unmapped and tokenized[0] == "*": continue
-        c = int(tokenized[2]) + int(tokenized[3]) # mapped + unmapped reads
-        perChromCount[tokenized[0]] = c
+    for chromosome in chromosomes: 
+        c = int(pysam.view('-F', '2304', '-c', bam, chromosome))
```
Comment (quejebo) [inline zippy/downsampling_bloom.py:153]: Can you add a comment about what sort of reads this covers?

**Comment by quejebo** [comment]
One small inline comment.  When that's fixed, everything else lgtm

**Comment by wukevin** [comment]
Same comment as @quejebo

**Comment by wukevin** [comment]
LGTM

**Pull request merged**